platforms/gke/base/use-cases/inference-ref-arch/online-inference-gpu/README.md

```diff
@@ -0,0 +1,224 @@
+    bucket where the model will be downloaded in
+    `platforms/gke/base/use-cases/inference-ref-arch/kubernetes-manifests/base/model-download.env`:
+
+    ```shell
+    IRA_BUCKET_NAME=<IRA_BUCKET_NAME>
+    MODEL_ID=<MODEL_ID>
+    ```
+
+    Where:
+
+    - `<IRA_BUCKET_NAME>` is the name of the Cloud Storage bucket where the
+      model will be downloaded.
+    - `MODEL_ID>` is the fully qualified model identifier.
+
+      - For Gemma, the fully qualified model identifier is:
+        `google/gemma-3-27b-it`
+      - For Llama 4, the fully qualified model identifier is:
+        `meta-llama/Llama-4-Scout-17B-16E-Instruct`
+      - For Llama 3.3, the fully qualified model identifier is:
+        `meta-llama/Llama-3.3-70B-Instruct`
+
+1.  [Generate a Hugging Face token](https://huggingface.co/docs/hub/security-tokens).
+    Make sure to grant the
+    `Read access to contents of all public gated repos you can access`
+    permission to the Hugging Face token.
+
+1.  Store the Hugging Face token in
+    `platforms/gke/base/use-cases/inference-ref-arch/kubernetes-manifests/base/hugging-face-token.env`:
+
+    ```shell
+    HUGGING_FACE_TOKEN=<HUGGING_FACE_TOKEN>
+    ```
+
+    Where:
+
+    - `<HUGGING_FACE_TOKEN>` is the Hugging Face token.
+
+    If the
+    `platforms/gke/base/use-cases/inference-ref-arch/kubernetes-manifests/base/hugging-face-token.env`
+    file doesn't exist, create it.
+
+1.  Get access to the model by signing the consent agreement:
+
+    - For Gemma:
+
+      1. Access the
+         [model consent page on Kaggle.com](https://www.kaggle.com/models/google/gemma).
+
+      1. Verify consent using your Hugging Face account.
+
+      1. Accept the model terms.
+
+    - For Llama:
+
+      1. Accept the model terms on Hugging Face
+
+1.  Deploy the model downloader in the GKE cluster:
+
+    ```shell
+    kubectl apply -k platforms/gke/base/use-cases/inference-ref-arch/kubernetes-manifests/model-download
+    ```
+
+1.  Wait for the model downloader to download the model:
+
+    ```shell
+    watch --color --interval 5 --no-title \
+      "kubectl get job/transfer-model-to-gcs | GREP_COLORS='mt=01;92' egrep --color=always -e '^' -e 'Complete'"
+    ```
+
+    The output is similar to the following:
+
+    ```text
+    NAME                    STATUS     COMPLETIONS   DURATION   AGE
+    transfer-model-to-gcs   Complete   1/1           33m        3h30m
+    ```
+
+## Deploy the online inference workload
+
+1.  Deploy the online inference workload in the GKE cluster:
+
+    ```shell
+    kubectl apply -k platforms/gke/base/use-cases/inference-ref-arch/kubernetes-manifests/online-inference-gpu-<GPU_TYPE>
+    ```
+
+    Where:
+
+    - `<GPU_TYPE>` is the GPU type. Valid values are:
+
+      - `h100`
+
+    - `MODEL_NAME` is the name of the model to deploy. Valid values are:
+
+      - `llama4-scout`
+
+    Ensure that you have enough quota in your project to provision the selected
```

Comment:
We should link to a page to teach the user on how to check the quota, an teach them how to check for specific GPU types